platform/ios/src/MGLAnnotationView.mm

```diff
@@ -19,23 +19,27 @@ @interface MGLAnnotationView () <UIGestureRecognizerDelegate>
 
 @implementation MGLAnnotationView
 
-- (instancetype)initWithReuseIdentifier:(NSString *)reuseIdentifier
-{
-    self = [self initWithFrame:CGRectZero];
++ (BOOL)supportsSecureCoding {
+    return YES;
+}
+
+- (instancetype)initWithReuseIdentifier:(NSString *)reuseIdentifier {
+    return [self initWithAnnotation:nil reuseIdentifier:reuseIdentifier];
+}
+
+- (instancetype)initWithAnnotation:(nullable id<MGLAnnotation>)annotation reuseIdentifier:(nullable NSString *)reuseIdentifier {
+    self = [super initWithFrame:CGRectZero];
```

Comment:
Note: changing this to `super` from the previous `self` fixes a runtime exception in Swift apps that subclass `MGLAnnotationView` without adding `initWithFrame`